### ISSUE TITLE
Max. work time of a driver 

### ISSUE THREAD

**Issue by ciblinek** [graphhopper/jsprit#99]
Hello,

according the law it is possible work just limited amount of time and after that time must be a break.
According to Czech (I guess also EU) law is possible work max. 4,5 hours in a row and after that must be 45 min in a row break.
In the real life is possible use 45 min like a break during waiting period before customer open.
Here is an example: Driver starts at 5 am that means that break must be at the latest at 9:30 am
but when arrives at 8 to customer who open at 8:45 these 45 min could be possible use as a break and it is
not necessary put next 45 min at 9:30 am. It is also possible split break to 15 + 30 min so if starts at 5 am and waiting time 1 is 15min
and waiting time 2 is 30min there is not necessary put next break at 9:30.

It would be nice if this constrains could be part of JSPRIT solution.
I hope that I explained it well if not please ask.

Thank you
Jan

**Comment by renanmzmendes** [comment]
Doing a quick search I couldn't find if breaks are supported by `jsprit`'s current version. Has there been a pull request to solve this?

**Comment by oblonski** [comment]
No, jsprit does not support breaks. It is on my ever growing to-implement-list.

**Comment by Rzpeg** [comment]
@oblonski This is really important due to EU Regulations.
Relates to #211
